transformations/tests/test_single_column_coalesced.py

```diff
@@ -222,59 +222,6 @@ def test_scc_base_masked_statement(frontend, horizontal):
     assert fgen(kernel_conds[0].else_body) == 'q(jl, jk) = t(jl, jk)'
 
 
-@pytest.mark.parametrize('frontend', available_frontends())
```

Comment:
Just to make sure we're not loosing test coverage: The deleted tests were essentially redundant in the sense that they tested functionality with the wrapper that's already been checked on the invidividual relevant transformations?

Reply:
Yes, I believe so. @awnawab might be better placed to confirm this, but I've glanced over them and they looked fairly identical still.

Reply:
Yup, all the tests with 'wrapper' in the name were essentially redundant.